app/models/users_event.rb

```diff
@@ -0,0 +1,4 @@
+class UsersEvent < ApplicationRecord
+  belongs_to :attended_event, class_name: 'Event', foreign_key: 'event_id'
+  belongs_to :attendee, class_name: 'User', foreign_key: 'user_id'
```

Comment:
Since the class name is the same as the foreign key, there is no need to specify it. You can leave it like this:
`belongs_to :attended_event, class_name: 'Event'`
 `belongs_to :attendee, class_name: 'User'`

Reply:
The foreign keys are not named the same as the class_name, so we needed to specify which foreign key rails needs to use.